lib/page/term_table.rb

```diff
@@ -49,7 +49,7 @@ def csv_url
     end
 
     def popolo_url
-      popolo_file.url
+      house.popolo_url
     end
```

Comment:
Looks like this method isn't being used in the term table view anymore, so probably best to just remove it.